edb/schema/delta.py

```diff
@@ -1399,6 +1399,8 @@ def _get_ast(self, schema, context):
             value = qlast.Tuple(elements=[
                 qlast.BaseConstant.from_python(el) for el in value
             ])
+        elif value is None:
```

Comment:
The `if new_value_empty` check above should handle this.  Remove ` and old_value_empty`, it's wrong.

Reply:
OK, I've removed the `value` check in favor of `new_value_empty`. As a result of this `old_value_empty` is no longer used anywhere so I dropped it.